ethpm_types/abi.py

```diff
@@ -13,6 +15,9 @@ class ABIType(BaseModel):
     type: Union[str, "ABIType"]
     internalType: Optional[str] = None
```

Comment:
Now that extras are allowed, curious if this field is even needed anymore. It's an extraneous field that Solidity sometimes injects, and not really necessary for our purposes here

Reply:
Note that removing this field wouldn't be a breaking change because nothing downstream relies on it

Reply:
Yeah I think it is fine to remove then.
The value still shows up in `dict()` so solidity still has the data. The extra field is still accessible on the class too, just doesn't know about it ahead of time.